test/zepto.html

```diff
@@ -2175,6 +2175,14 @@
           $.map(el.children(), function(c){ return $(c).text() }).join(' '))
       },
 
+      testAppendWithArrayParams: function(t){
+        var node = document.createElement('b');
+        node.appendChild(document.createTextNode('four'))
+        var el = $('<div></div>')
+        el.append(['<b>one</b><b>two</b>', '<b>three</b>', node])
+        t.assertEqual('<b>one</b><b>two</b><b>three</b><b>four</b>', el.html())
```

Comment:
Hmm, now that I read jQuery documentation, it doesn't seem that it's documented that the elements within this array can be HTML strings or jQuery objects:

> `content`: DOM element, **array of elements**, HTML string, or jQuery object to insert at the end of each element in the set of matched elements.